src/main/java/net/geforcemods/securitycraft/entity/SentryEntity.java

```diff
@@ -376,7 +402,7 @@ public void setDisguiseModule(ItemStack module)
 			ItemStack disguiseStack = blocks.get(0);
 			BlockState state = Block.getBlockFromItem(disguiseStack.getItem()).getDefaultState();
 
-			world.setBlockState(getPosition(), state.getShape(world, getPosition()) == VoxelShapes.fullCube() ? state : Blocks.AIR.getDefaultState());
+			if (world.getBlockState(getPosition()) == Blocks.AIR.getDefaultState()) world.setBlockState(getPosition(), state.getShape(world, getPosition()) == VoxelShapes.fullCube() ? state : Blocks.AIR.getDefaultState());

```

Comment:
1. New line after the if condition.
2. Check for `world.getBlockState.isAir` instead of directly comparing with `Blocks.AIR`.